tests/transformer/test_assign.py

```diff
@@ -54,6 +50,6 @@ def test_RestrictingNodeTransformer__visit_Assign__2(
     assert glb['e'] == [3, 4]
     assert glb['x'] == 5
 
-    _getiter_.assert_has_calls([
-        mocker.call((1, 2, 3, (4, 3, 4), 5)),
-        mocker.call((4, 3, 4))])
+    _getiter_.assert_has_calls(
+        [mocker.call((1, 2, 3, (4, 3, 4), 5)), mocker.call((4, 3, 4))]
+    )
```

Comment:
It was more clear before because I could see easily that there are two calls because they were on two lines.